.vite/deps/_metadata.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Looks like folder .vite shouldn't be in git, please check if it works without it